shared/profile/user/container.tsx

```diff
@@ -49,7 +49,14 @@ const connected = Container.namedConnect(
     if (!notAUser) {
       // Keybase user
       const followThem = Constants.followThem(state, username)
-      const {followersCount, followingCount, followers, following, reason, webOfTrustEntries} = d
+      const {followersCount, followingCount, followers, following, reason} = d
+      const webOfTrustEntries = (d.webOfTrustEntries || []).filter(entry => Types.showableWotEntry(entry))
```

Comment:
1. This part is strongly suggested: `filter(Types.showableWotEntry)`
2. The default value assignment `webOfTrustEntries = []` is optional.

```suggestion
      const {followersCount, followingCount, followers, following, reason, webOfTrustEntries = []} = d
      const wotEntries = (webOfTrustEntries).filter(Types.showableWotEntry)
```

Reply:
Change `Types.showableWotEntry` to `Constants.showableWotEntry` per above comment

Reply:
👍